Competition2017/src/competition/subsystems/drive/DriveSubsystem.java

```diff
@@ -164,5 +177,26 @@ public void updateMotorTelemetry() {
         leftDriveSlave.updateTelemetryProperties();
         rightDrive.updateTelemetryProperties();
         rightDriveSlave.updateTelemetryProperties();
+        
+        leftDistanceProp.set(leftDrive.getPosition());
+        rightDistanceProp.set(rightDrive.getPosition());
+    }
+    
+    public double getDisplacement() {
+        /* TODO add more values from other motors for accuracy */
+        return convertTicksToInches(leftDrive.getPosition() - startPositionTicks.get());
+    }
+    
+    public void resetDistance() {
```

Comment:
let's call this resetDisplacement so it's clear they're talking about the same thing if we're going with displacement instead